iterfzf/__init__.py

```diff
@@ -45,6 +45,8 @@ def iterfzf(
         cmd.append('--exact')
     if multi:
         cmd.append('--multi')
+    if ansi:
+        cmd.append('--ansi')
```

Comment:
I'll move this down actually, just noticed the other blocks are in the same order as the arguments.